draft-ietf-quic-transport.md

```diff
@@ -2077,9 +2084,13 @@ genuine migrations.  Changing port number can cause a peer to reset its
 congestion state (see {{migration-cc}}), so the port SHOULD only be changed
 infrequently.
 
-An endpoint that exhausts available connection IDs cannot migrate.  To ensure
-that migration is possible and packets sent on different paths cannot be
-correlated, endpoints SHOULD provide new connection IDs before peers migrate.
+An endpoint that exhausts available connection IDs cannot migrate.  Similarly,
+an endpoint is unable to respond to probes or an attempt by its peer to migrate
+or probe.  To ensure that migration is possible and packets sent on different
+paths cannot be correlated, endpoints SHOULD provide new connection IDs before
+peers migrate.  If a peer might have exhausted available connection IDs, a
```

Comment:
Do we want to reference the other section that talks about SHOULD provide a new CID for every one that gets retired, etc.? I think that's `{{issue-cid}}`.

Specifically the below paragraph: 
```
An endpoint SHOULD supply a new connection ID when it receives a packet with a
previously unused connection ID or when the peer retires one, unless providing
the new connection ID would exceed the peer's limit.  An endpoint MAY limit the
frequency or the total number of connection IDs issued for each connection to
avoid the risk of running out of connection IDs; see {{reset-token}}.
```

```suggestion
peers migrate ({{issue-cid}}).  If a peer might have exhausted available connection IDs, a
```

If that doesn't mess up the spacing.